tests/test_download.py

```diff
@@ -129,6 +131,27 @@ def test_no_credentials_error(self, remote_local_file: Any):
             download_from_gcs(mock_remote_filepath, mock_local_filepath)
 
 
+class TestDatabricksUnityCatalog:
+
+    def test_invalid_prefix_from_db_uc(self, remote_local_file: Any):
+        with tempfile.NamedTemporaryFile(delete=True, suffix='.txt') as tmp:
+            file_name = tmp.name.split(os.sep)[-1]
+            mock_remote_filepath, _ = remote_local_file(cloud_prefix='dbfs:/Volumess',
```

Comment:
should the `cloud_prefix` here be `'dbfs:/Volumes'` with just one 's'?

Reply:
I intentionally kept it to hit the exception in case use made such a mistake, then we should throw a proper descriptive error message.

Reply:
ah nice